dotNetInstallerToolsLib/FileUtil.cpp

```diff
@@ -217,10 +217,13 @@ bool DVLib::ResourceExists(HMODULE h, const std::wstring& resource, const std::w
     return (NULL != ::FindResource(h, resource.c_str(), type.c_str()));
 }
 
-DVLib::FileVersion DVLib::wstring2fileversion(const std::wstring& version)
+DVLib::FileVersion DVLib::wstring2fileversion(std::wstring version)

```

Comment:
I don't love changing interfaces to not take constants and think I'd prefer the version being copied inside the function, but I can live with it.